index.md

```diff
@@ -120,12 +120,13 @@ about-peer-review/code-of-conduct
 :hidden:
 :caption: Peer Review Guides
 
-Guide intro <software-peer-review-guide/intro>
+Peer Review Timeline <software-peer-review-guide/intro>
```

Comment:
## TODO

somewhere in this guide i talk about how the presubmission process and editor checks by themselves can be a conversation with the author that improves packaging. provide some specific examples using the pyteny review that is open now as a presubmission!